sdk/schemaregistry/schema-registry-json/src/jsonSchemaSerializer.ts

```diff
@@ -114,7 +114,7 @@ export class JsonSchemaSerializer<MessageT = MessageContent> {
    * @throws {@link Error}
    * Thrown if the deserialization failed, e.g. because reader and writer schemas are incompatible.
    */
-  async deserialize(message: MessageT, options?: DeserializeOptions): Promise<unknown> {
+  async deserialize<T = unknown>(message: MessageT, options?: DeserializeOptions): Promise<T> {
```

Comment:
If you prefer to use generics in this context, you may leave it as required. If not provided, it will default to `unknown`. Additionally, if you find the need to include another required generic after T for any reason, you can still do that.

```suggestion
  async deserialize<T>(message: MessageT, options?: DeserializeOptions): Promise<T> {
```